go/vt/mysqlctl/mysqld.go

```diff
@@ -107,9 +112,92 @@ func NewMysqld(dbcfgs *dbconfigs.DBConfigs) *Mysqld {
 	result.appPool = dbconnpool.NewConnectionPool("AppConnPool", *appPoolSize, *appIdleTimeout, *poolDynamicHostnameResolution)
 	result.appPool.Open(dbcfgs.AppWithDB(), appMysqlStats)
 
+	version, getErr := getVersionString()
+	f, v, err := parseVersionString(version)
+
+	// Fallback if required
+	if getErr != nil || err != nil {
+		f, v, err = getVersionFromEnv()
+		if err != nil {
+			panic("Could not detect version from mysqld --version or MYSQL_FLAVOR")
+		}
+
+	}
+
+	log.Infof("Using flavor: %v, version: %v", f, v)
+	result.capabilities = NewCapabilitySet(f, v)
 	return result
 }
 
+func getVersionFromEnv() (flavor mysqlFlavor, ver serverVersion, err error) {
+
```

Comment:
Remove extra lines at top and bottom of function to be consistent with code style of the project.